frontend/src/app/main/ui/workspace/tokens/modals/themes.cljs

```diff
@@ -197,16 +198,26 @@
 
 (mf/defc theme-modal-buttons*
   [{:keys [close-modal on-save-form disabled?] :as props}]
-  [:*
-   [:> button* {:variant "secondary"
-                :type "button"
-                :on-click close-modal}
-    (tr "labels.cancel")]
-   [:> button* {:variant "primary"
-                :type "submit"
-                :on-click on-save-form
-                :disabled disabled?}
-    (tr "workspace.token.save-theme")]])
+  (let [handle-key-down
+        (mf/use-fn
```

Comment:
This use-fn (React.useCallback) has no real effect, because immediately afterwards you are calling it for generating new function instances.

You probably should define two separated handlers.